styles/components/_autocomplete.scss

```diff
@@ -0,0 +1,39 @@
+// ==================================================================
+// Autocomplete
+// ==================================================================
+
+////
+/// @group autocomplete
+////
+.sprk-c-Autocomplete__results {
+  list-style-type: none;
```

Comment:
I think that `list-style-type` and `position` should still have variables, but we can set `themable` to false, if necessary.